src/utils/ValidationHelper.ts

```diff
@@ -36,8 +36,7 @@ export class ValidationHelper {
 		if (headerValue === null || headerValue === undefined) {
 			throw new AppError("Missing header: Authorization header value is missing or invalid auth_scheme", HttpCodesEnum.UNAUTHORIZED);
 		}
-		const authHeader = event.headers.Authorization as string;
-		if (authHeader !== null && !authHeader.includes(Constants.BEARER)) {
+		if (!headerValue.includes(Constants.BEARER + " ")) {
```

Comment:
This caused an API test to break in F2F - have we confirmed this it doesn't do the same in CIC?

Reply:
Yeah i have verified the same, there isnt any similar test for this that i could find. Also Zac has ran the tests on this branch and all looks good, attached the result to the ticket.